steps/list-jira-tickets/0.0.1/step.yml

```diff
@@ -0,0 +1,69 @@
+title: JIRA tickets listing
+summary: |
+  Lists all JIRA tickets by comparing two branches.
+description: |
+  Lists all JIRA tickets by comparing two branches. It uses the JIRA ticket prefix to parse the commit messages. It will output either a list of ticket IDs or links if you add the URL path.
+website: https://bitbucket.org/silofit/bitrise-jira-tickets/src/master/
+source_code_url: https://bitbucket.org/silofit/bitrise-jira-tickets/src/master/
+support_url: https://bitbucket.org/silofit/bitrise-jira-tickets/src/master/
+published_at: 2021-08-17T15:09:12.991958-04:00
+source:
+  git: https://amarliac@bitbucket.org/silofit/bitrise-jira-tickets.git
+  commit: 71564b541d2d52a93e87b06bfe075abd219fdf84
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  apt_get:
+  - name: git
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
+    description: |
+      This is the branch you will be listing the JIRA tickets from.
+    is_expand: true
+    is_required: true
+    summary: null
+    title: Selected Git branch
+  selected_branch: null
+- opts:
+    description: |
+      This is the branch you will be comparing your selected branch with to list the JIRA tickets.
+    is_expand: true
+    is_required: true
+    summary: null
+    title: Reference Git branch
+  reference_branch: null
+- opts:
+    description: |
+      Adding the URL path of your JIRA workspace will output the ticket links instead of just a list of IDs.
+    is_expand: true
+    is_required: false
+    summary: null
+    title: JIRA ticket's URL path
+  url_path: null
+- opts:
+    description: |
+      JIRA ticket's prefix is necessary to extract the tickets when parsing the commit messages. It is following the pattern: [prefix]-[ticket number]. Example: "JTS" as a prefix will extract JIRA tickets like "JTS-1234"
+    is_expand: true
+    is_required: true
+    summary: null
+    title: JIRA ticket's prefix
+  prefix: null
+outputs:
+- JIRA_TICKET_IDS: null
```

Comment:
NOTE: having `_LIST` suffix for outputs with multiple values is the idiomatic way: https://devcenter.bitrise.io/contributors/create-your-own-step/#outputs-with-list-of-values

Reply:
Addressed